docs/resources/helm_feature.md

```diff
@@ -0,0 +1,124 @@
+---
+Title: "Helm Feature Resource"
+Description: |-
+    Creating the Helm Feature resource.
+---
+
+# Helm Feature
+
+The `tanzu-mission-control_helm_feature` resource allows you to enable and disable helm feature to a particular scope through Tanzu Mission Control.
+
+When you enable the Helm service on a cluster or cluster group, you can then create releases in your cluster from Helm charts stored in the Bitnami repository.
+To enable a helm service, you must be associated with the cluster.admin or clustergroup.admin role.
+
+[helm-feature]: https://docs.vmware.com/en/VMware-Tanzu-Mission-Control/services/tanzumc-using/GUID-0927CDC8-A5C1-4FAE-9A7C-8A5D62FDF8D8.html
+## Helm Feature Scope
+
+In the Tanzu Mission Control resource hierarchy, there are two levels at which you can specify Helm Feature resources:
+- **object groups** - `cluster_group` block under `scope` sub-resource
+- **Kubernetes objects** - `cluster` block under `scope` sub-resource
+
+**Note:**
+The scope parameter is mandatory in the schema and the user needs to add one of the defined scopes to the script for the provider to function.
+Only one scope per resource is allowed.
+
+## Cluster group scoped Helm Feature
+
+### Example Usage
+
+```terraform
+# Create Tanzu Mission Control cluster group scope helm feature with attached set as default value.
+resource "tanzu-mission-control_helm_feature" "create_cg_helm_feature" {
+  scope {
+    cluster_group {
+      name = "default" # Required
+    }
+  }
+
+  meta {
+    description = "Create namespace through terraform"
+    labels      = { "key" : "value" }
+  }
+}
+```
+
+## Cluster scoped Helm Feature
+
+### Example Usage
+
+```terraform
+# Create Tanzu Mission Control cluster scope helm feature with attached set as default value.
+resource "tanzu-mission-control_helm_feature" "create_cl_helm_feature" {
+  scope {
+    cluster {
+      name            = "testcluster" # Required
+      provisioner_name        = "attached"    # Default: attached
+      management_cluster_name = "attached"    # Default: attached
+    }
+  }
+
+  meta {
+    description = "Create namespace through terraform"
+    labels      = { "key" : "value" }
+  }
+}
+```
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `scope` (Block List, Min: 1, Max: 1) Scope for the Helm feature, having one of the valid scopes: cluster, cluster_group. (see [below for nested schema](#nestedblock--scope))
+
+### Optional
+
+- `meta` (Block List, Max: 1) Metadata for the resource (see [below for nested schema](#nestedblock--meta))
+
+### Read-Only
+
+- `id` (String) The ID of this resource.
+- `status` (Map of String) Status for the Repository.
```

Comment:
Should it be phase?

Reply:
it's at the Upper level of the schema, so it should be status.